kafka_consumer/datadog_checks/kafka_consumer/client.py

```diff
@@ -121,13 +123,14 @@ def get_consumer_offsets(self):
 
         consumer_groups = self._get_consumer_groups()
 
-        for future in self._get_consumer_offset_futures(consumer_groups):
+        futures = self._get_consumer_offset_futures(consumer_groups)
+
+        for future in as_completed(futures):
             try:
                 response_offset_info = future.result()
             except KafkaException as e:
                 self.log.debug("Failed to read consumer offsets for future %s: %s", future, e)
             else:
-                self.log.debug('FUTURE RESULT: %s', response_offset_info)
```

Comment:
non blocking nit:

What was the reasoning for getting rid of this log line?

Reply:
This output wasn't all that useful, since it prints out the object rather than the contents of the object (which is logged later down the function).